pkg/gadgets/internal/networktracer/tracer.go

```diff
@@ -116,6 +117,9 @@ func (t *Tracer[Event]) newAttachment(
 		MapReplacements: map[string]*ebpf.Map{
 			"tail_call": t.dispatcherMap,
 		},
+		Programs: ebpf.ProgramOptions{
+			KernelTypes: btfgen.GetBTFSpec(),
```

Comment:
The dispatcher program is a socket filter that only uses the stable Linux API, it should need BTF.

I am worried that setting `KernelTypes` would slow down the performances unnecessarily.

Reply:
Makes sense. I'll remove custom BTF for this program.